siriuspy/siriuspy/devices/device.py

```diff
@@ -179,7 +179,21 @@ def pv_attribute_values(self, attribute):
         return attributes
 
     def wait(self, propty, value, timeout=None, comp='eq'):
-        """."""
+        """Wait until comparison of property value is true against 'value'.
+
+        Args:
+            propty (str): Name of the property to wait condition to be met.
+            value (str|float|int|bool|numpy.ndarray): value to be set.
+            timeout (float, optional): Timeout of operation. Defaults to None,
```

Comment:
The code accepts `never` for `timeout` as well. Docstring restricts to `float`

Reply:
I prefer to track this in repository issues. Just a suggestion.

Reply:
It's a good idea. I'll add a card to our Trello not to forget to create the issue! 🤣